Sources/PointFree/Stripe.swift

```diff
@@ -189,7 +204,7 @@ extension Tagged where Tag == Stripe.Plan, A == String {
 //  .map(^\.data >>> filter(StripeSubscriptionPlan.Id.all.contains <<< ^\.id))
 
 private func cancelSubscription(id: Stripe.Subscription.Id) -> EitherIO<Prelude.Unit, Stripe.Subscription> {
-  return stripeDataTask("subscriptions/\(id.rawValue)", .delete)
+  return stripeDataTask("subscriptions/" + id.rawValue, .delete(["at_period_end": "true"]))
```

Comment:
are you just leaning away from interpolation or is there another reason for this?

Reply:
Yeah, leaning away to avoid bugs.